.env.example

```diff
@@ -68,6 +68,13 @@ VUE_APP_HOME_FEATURES_COLUMNS=
 # Allowed values: 'ecommerce' and 'service' (default value)
 VUE_APP_MARKETPLACE_TYPE=
 
+# Only for ecommerce marketplaces (`VUE_APP_MARKETPLACE_TYPE=ecommerce`)
+# Provide comma-separated fees’ types that will be used (e.g. 'deliveryFee')
+# to display amounts in price table
+# They will be retrieved via owner metadata (`owner.metadata.instant.deliveryFee`)
+# and applied once per order
+VUE_APP_ORDER_FEE_TYPES=
```

Comment:
Why do you need this variable?

I don’t get the purpose of comma-separated values if you apply this `once per order`. 

We agreed on slack to let users customize delivery fees and just refactor order summary table to allow custom lines.

Reply:
I added this variable to be able to apply one or several types of fees per order.
Order summary table takes into account custom lines. And this variable is useful to let takers to preview the total price before payment.

I forgot to integrate an input to let owner enter the delivery fee. I'll add it, thanks.

Reply:
You’re not answering my question.

What are the expected types, and why not just detect `deliveryFee` and add it to the table?

Reply:
The types can be any string specified by developers.

I'll add an example for the big picture.
Let's suppose a platform need two types of fees for each order : `deliveryFee` and `countryTaxFee` (arbitrary string).
We fill the value in `.env.example`:
```js
VUE_APP_ORDER_FEE_TYPES=deliveryFee,countryTaxFee
```

By specifying the order fee types, the content must be added in the file `theme/pricing.yml`:
https://github.com/stelace/marketplace-demo/blob/0813d49acf04f6977af1564827610fd1b33980ad/src/i18n/theme/pricing.yaml#L1-L12
`deliveryFee` keys are already in the content file. For this case, `countryTaxFee_label` and `countryTaxFee_with_price` must be added.

The specified order type fees will also be used in the price table before payment ([TransactionCard](https://github.com/stelace/marketplace-demo/blob/0813d49acf04f6977af1564827610fd1b33980ad/src/components/TransactionCard.vue#L382-L407), [Cart](https://github.com/stelace/marketplace-demo/blob/0813d49acf04f6977af1564827610fd1b33980ad/src/pages/Cart.vue#L139-L164) components) and in the [Netlify function to create order](https://github.com/stelace/marketplace-demo/blob/0813d49acf04f6977af1564827610fd1b33980ad/netlify/functions/createOrder.js#L74-L90) based on instant metadata properties detected in the owner object.

Only detecting the property `deliveryFee` isn't enough to display the delivery fee input to owner,
because we cannot distinguish if the platform doesn't want delivery fee or if there is no value yet.

In fact, `VUE_APP_ORDER_FEE_TYPES=string1,string2,...` is a generalization of `VUE_APP_DELIVERY_FEE_ACTIVE=boolean`.